include/mbedtls/psa_util.h

```diff
@@ -127,10 +132,23 @@ psa_ecc_family_t mbedtls_ecc_group_to_psa(mbedtls_ecp_group_id grpid,
  *                      (`PSA_ECC_FAMILY_xxx`).
  * \param bits          The bit-length of a private key on \p curve.
  *
- * \return              The corresponding Mbed TLS elliptic curve identifier
- *                      (`MBEDTLS_ECP_DP_xxx`).
+ * \return              On success the corresponding Mbed TLS elliptic curve
+ *                      identifier (`MBEDTLS_ECP_DP_xxx`).
  * \return              #MBEDTLS_ECP_DP_NONE if the combination of \c curve
- *                      and \p bits is not recognized.
+ *                      and \p bits is not supported or invalid:
+ *                      - not supported means that the proper `PSA_WANT_ECC_xxx`
+ *                        symbol is not enabled for the requested curve.
+ *                      - invalid if `PSA_WANT_ECC_xxx` is enabled, but the
+ *                        combination of \p curve and \p bits are not correct
+ *                        for that curve.
+ * \return              #MBEDTLS_ECP_DP_NONE for secp224k1 curve, no matter
```

Comment:
I don't think this sentence is useful at all here. It's covered by the statement above. It just happens that `PSA_WANT_ECC_SECP_K_1` cannot be enabled.

(This statement is useful in the implementation, to explain to maintainers why the case for secp244k1 is missing where all the other curves supported by Mbed TLS are present.)